ansible/roles/proxy/tasks/main.yml

```diff
@@ -0,0 +1,63 @@
+- name: Define configuration in /etc/environment
+  tags: proxy
+  lineinfile:
+    path: "/etc/environment"
+    create: yes
+    owner: root
+    group: root
+    mode: o=rw,go=r
+    state: present
+    regexp: "{{ item.key }}=.*"
+    line: "{{ item.key }}={{ item.value }}"
+  loop:
+    - key: http_proxy
+      value: "{{ proxy_http_proxy }}"
+    - key: https_proxy
+      value: "{{ proxy_https_proxy }}"
+    - key: no_proxy
+      value: "{{ proxy_no_proxy }}"
+
+- name: Define dnf/yum proxy
+  ini_file:
+    path: /etc/yum.conf
```

Comment:
Should this be `/etc/dnf/dnf.conf`?

Reply:
oops, yes! /etc/yum.conf is normally symlinked to /etc/dnf/dnf.conf; apparently changing the former works.